website/docs/Installing.mdx

```diff
@@ -42,9 +42,11 @@ import App from "./App";
 +   Navigation.setRoot({
 +     root: {
 +       stack: {
-+         component: {
-+           name: 'com.myApp.WelcomeScreen'
-+         }
++         children: [
++           component: {
++             name: 'com.myApp.WelcomeScreen'
++           }
++         ]
```

Comment:
Stack needs to have `children` unless we wanted to do:

```js
root: {
  component: {... }
}
```

Reply:
Great catch! This is my mistake 😅